libs/extensions/angular/image-banner/src/image-banner.component.stories.ts

```diff
@@ -0,0 +1,102 @@
+import { argsToTemplate, type Meta, type StoryObj } from '@storybook/angular';
+import { ImageBannerComponent } from '@kirbydesign/extensions-angular/image-banner';
+
+/**
+ * TODO: Awaiting banner description from UX guidelines.
```

Comment:
TODO: Add before merging. 
@henrikvoetmand do you know if we have a description?